File: apps/auth-server/src/main/java/com/example/chat/auth/server/config/security/JwtProperties.java
package com.example.chat.auth.server.config.security;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

import lombok.Getter;
import lombok.Setter;

/**
 * JWT 설정 Properties
 */
@Configuration
@ConfigurationProperties(prefix = "jwt")
@Getter
@Setter
public class JwtProperties {
    private String issuer = "http://localhost:18080";
    private long accessTokenExpiration = 3600L;
    private long refreshTokenExpiration = 86400L;
}

File: apps/auth-server/src/main/kotlin/com/example/chat/auth/server/config/security/JwtProperties.kt

package com.example.chat.auth.server.config.security

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * JWT 설정 Properties
 */
@ConfigurationProperties(prefix = "jwt")
data class JwtProperties(
    var issuer: String = "http://localhost:18080",
    var accessTokenExpiration: Long = 3600L,
    var refreshTokenExpiration: Long = 86400L
)
